app/views/layouts/application.html.erb

```diff
@@ -47,3 +47,18 @@
     });
 <% end %>
 </script>
+
+<% if Rails.env.development? || Rails.env.staging? %>
+  <script type="text/javascript">
+    var bugmuncher_options = {
+      api_key: "fe678532af029a3c85a2"
```

Comment:
Yeah that's what I was thinking tbh, let's see if it works for us for a few days?